account_analytic_security/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Account Analityc Security",
+    "summary": """
+        Add a new group and two registration rules to restrict users from creating or deleting analytical accounts
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "15.0.1.0.0",
+    'category': "Accounting & Finance",
+    "website": "https://github.com/solvosci/slv-account",
+    "depends": ["account"],
```

Comment:
Is this dependency the minor one that must be required? Please check

Reply:
depending of analytics also works but this warning appears: 
![imagen](https://github.com/user-attachments/assets/3d4988cd-299f-48e5-bf4c-15f69028619d)